optaplanner-examples/src/main/java/org/optaplanner/examples/scrabble/domain/ScrabbleCell.java

```diff
@@ -36,7 +36,7 @@
 
     @CustomShadowVariable(variableListenerClass = CellUpdatingVariableListener.class,
             sources = {@PlanningVariableReference(entityClass = ScrabbleWordAssignment.class, variableName = "startCell"),
-                    @PlanningVariableReference(entityClass = ScrabbleWordAssignment.class, variableName = "direction"),})
+                       @PlanningVariableReference(entityClass = ScrabbleWordAssignment.class, variableName = "direction")})
```

Comment:
I'll remove this change, but for the future, should it be 4 for the CustomShadowVariable annotation, 8 for the `sources =` line, and 12 for the second `PlanningVariableReference` line? Or 4 / 8 / 16?